components/src/core/drawer/drawer.tsx

```diff
@@ -32,17 +34,19 @@ export type DrawerProps = ViewProps &
          * Overrides for theme
          */
         theme?: $DeepPartial<ReactNativePaper.Theme>;
+        // Whether the drawer is open
+        open?: boolean;
```

Comment:
what about a permanent drawer on a tablet? Or is that for the future?

Reply:
I think we can consider that for the future...it would involve building an entirely new variant / refactoring some of how the drawer currently works (we'd have to build our own expand collapse mechanism, since react-navigation won't keep the drawer on the screen like that for us, i don't think).